queue/src/fiskaltrust.Middleware.Localization.QueueAT/RequestCommands/InitialOperationReceiptCommand.cs

```diff
@@ -114,7 +125,7 @@ internal async Task<(List<ftActionJournal> actionJournals, ftJournalAT journalAT
             actionJournals.Add(aj);
 
 
-            var (receiptIdentification, ftStateData, isBackupScuUsed, signatureItems, journalAT) = await SignReceiptAsync(queueAT, request, response.ftReceiptIdentification, response.ftReceiptMoment, queueItem.ftQueueItemId);
+            var (receiptIdentification, ftStateData, isBackupScuUsed, signatureItems, journalAT) = await SignReceiptAsync(queueAT, request, response.ftReceiptIdentification, response.ftReceiptMoment, queueItem.ftQueueItemId, true);
```

Comment:
I personally think that named parameters for boolean flags at a lot of readability quality, e.g. here:

```suggestion
            var (receiptIdentification, ftStateData, isBackupScuUsed, signatureItems, journalAT) = await SignReceiptAsync(queueAT, request, response.ftReceiptIdentification, response.ftReceiptMoment, queueItem.ftQueueItemId, isZeroReceipt: true);
```